### PR TITLE
Feature/default img

### LINKED ISSUE #74: [TO IMPLEMENT] default.jpg
When Google and 42 don't send images there should be a default.
We should add it to uploads folder.

### PR DESCRIPTION
**Pull request by J0Santos**
If no image is found for any profile, Marvin's default image will be displayed.

### CODE REVIEW

File: backend/src/db_interactions_modules/users/users.service.ts [hunk 1]
```diff
@@ -45,15 +45,26 @@ export class UsersService {
...
+   	   nick: nick_
+   	 }});
+   	 // console.log(resp)
+   	 if(!resp) {
+   	  return res.status(404).json()
+		 }
+   	 else {
+      let fileName = './uploads/' + resp.avatar.split('/').pop();
+			if (!resp.avatar || !existsSync(fileName)) {
+				resp.avatar = 'http://localhost:3000/users/avatar/default.jpg';
```
Comment (ncameiri42) [inline backend/src/db_interactions_modules/users/users.service.ts]:
```suggestion
				resp.avatar = process.env.BACKEND_URL + '/users/avatar/default.jpg';
```

File: backend/src/db_interactions_modules/users/users.service.ts [hunk 2]
```diff
@@ -91,7 +102,15 @@ export class UsersService {
     const resp= await this.userRepository.findOne(
       {where: {id: id_to_search}}
      );
-     return resp;
+		 if(!resp) {
+			return null
+		}
+    let fileName = './uploads/' + resp.avatar.split('/').pop();
+		if (!resp.avatar || !existsSync(fileName)) {
+		 resp.avatar = 'http://localhost:3000/users/avatar/default.jpg';
```
Comment (ncameiri42) [inline backend/src/db_interactions_modules/users/users.service.ts]:
```suggestion
		 resp.avatar = process.env.BACKEND_URL + '/users/avatar/default.jpg';
```

**Review by ncameiri42 — COMMENTED** [review]
please use Backend_URL env variable instead of localhost, otherwise on evaluation will fail using some 10.X.X.X ip style to use our App

**Review by gleal42 — APPROVED** [review]
LGTM!

**Pull request merged**